Updates to History Limits (#2013)
temporalio/documentation: docs-src/concepts/what-is-an-event-history.md

The Temporal Cluster stores the complete Event History for the entire lifecycle of a Workflow Execution.

> The Temporal Cluster logs a [warning after 10K (10,240) Events](/workflows#limits) and will periodically log additional warnings as new Events are added.
> If the Event History exceeds 50K (51,200) Events, the Workflow Execution will be terminated.

Reviewer: ```suggestion
The Temporal Cluster logs a [warning after 10K (10,240) Events](/workflows#limits) and periodically logs additional warnings as new Events are added.
If the Event History exceeds 50K (51,200) Events, the Workflow Execution is terminated.
```
